ui/pages/add-token/add-token.component.js

```diff
@@ -358,15 +366,16 @@ class AddToken extends Component {
   }
 
   renderSearchToken() {
+    const { tokenList = {} } = this.props;
```

Comment:
It might be cleaner if we use `AddToken .defaultProps` to set this `tokenList` value to empty object.